sdks/go/pkg/beam/io/textio/textio_test.go

```diff
@@ -40,3 +46,68 @@ func TestRead(t *testing.T) {
 	}
 
 }
+
+func TestRead(t *testing.T) {
+	p, s := beam.NewPipelineWithRoot()
+	lines := Read(s, testFilePath)
+	passert.Count(s, lines, "NumLines", 1)
+
+	if _, err := beam.Run(context.Background(), "direct", p); err != nil {
```

Comment:
Please change these to use the ptest helpers for executing pipeline instead of the normal infrastructure, and explicitly leaning on the direct runner. In particular: [ptest.RunAndValidate](https://pkg.go.dev/github.com/apache/beam/sdks/v2@v2.36.0/go/pkg/beam/testing/ptest#RunAndValidate)